src/main/scad/yavuz-demirhan/Raya1.scad

```diff
@@ -0,0 +1,66 @@
+include <puzzlecad2.scad>
+
+box_puzzle_scale = 15;
+box_puzzle_border = 6;
+$burr_inset = 0.16; 
+long = box_puzzle_scale * 4; // voxels long
+wide = box_puzzle_scale * 3; // voxels wide
+high = box_puzzle_scale * 2; // voxels high
+
+dimlong = long + box_puzzle_border * 2;
+dimwide = wide + box_puzzle_border * 2;
+height = high + box_puzzle_border;
```

Comment:
If you follow my suggestion above, this can become

dim = interior_dim + [box_puzzle_border * 2, box_puzzle_border * 2, box_puzzle_border];

Then dim.x, dim.y, dim.z will refer to the outer dimensions.

Reply:
Cool, I did not know dimensions could be done as vectors. This is much better.